action.yml

```diff
@@ -28,13 +30,18 @@ runs:
     shell: bash
   - run: |
       import Pkg
+      
+      # Determine if Pkg uses git-cli executable instead of LibGit2
```

Comment:
Might be good to have a warning or at least info message if the input is set but the version is <= 1.6. Perhaps a warning is too much noise but at least an info that the input isn't being used is probably good?

Reply:
Good point! Added it as a yellow print statement, so that it should catch the eye more easily :+1: